index.yaml

```diff
@@ -900,7 +921,7 @@ entries:
     version: 6.11.0
   - apiVersion: v2
     appVersion: "1.0"
-    created: "2024-01-09T17:18:19.336479925Z"
+    created: "2024-02-21T14:04:56.154085-08:00"
```

Comment:
This looks wrong. The digest for 6.11 should not change.

Reply:
I had actually changed the digest here, too. I left the comment on a line that doesn't show the context.

Reply:
https://github.com/lacework/helm-charts/commit/4e3ecaaa94fbdf80d5790ece97c96954b2b090c4#r139131323